src/pages/utilities/UnrecognizedUtilityTabs/UnrecognizedTab.tsx

```diff
@@ -76,7 +70,27 @@ const Menu = (
   const tableSelectedRows = table.getSelectedRowModel();
   const selectedRows = useMemo(() => tableSelectedRows.rows.map(row => row.original), [tableSelectedRows]);
 
-  const deleteFiles = () => {
+  const showDeleteConfirmation = useEventCallback(() => {
+    setShowConfirmModal(true);
+  });
+
+  const cancelDelete = useEventCallback(() => {
+    setShowConfirmModal(false);
+  });
+
+  const removeFileFromSelection = useEventCallback((fileId: number) => {
+    // Figure this out.
+    some(tableSelectedRows.rows, (row) => {
+      if (row.original.ID === fileId) {
+        row.toggleSelected(false);
+        return true;
+      }
+      return false;
+    });
+  });
```

Comment:
I'm all for using new library functions but a `forEach` and `break` is more easily understandable by majority of the people

Reply:
In this case, some is a short circuited forEach but if you return false from normal forEach it is same behavior, I usually use forEach with return false.

Reply:
Also it only deletes 1 file it seems, is that the intent?

Reply:
It removes the file from the list, it doesn't delete it. Also didn't know about the fact that we can `return false` in a `forEach` loop (in lodash) to get it to terminate early.